runtime/src/main/scala/akka/grpc/GrpcClientSettings.scala

```diff
@@ -21,16 +21,25 @@ import scala.concurrent.duration.{ Duration, _ }
 
 object GrpcClientSettings {
 
-  /** Scala API */
+  /**
+   * Scala API
+   *
+   * Create a client hat uses a static host and port. Does not load
+   * any default configuration from reference.conf
+   */
   def apply(host: String, port: Int): GrpcClientSettings =
```

Comment:
I wonder if we should really keep this, maybe we should always require either an ActorSystem or a config so that we can load defaults. I think a few of the settings in Akka and Akka HTTP work like that.

Reply:
Should we make it implicit?